openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java

```diff
@@ -359,7 +337,7 @@ private List<GenSrcFile> generateClientFiles(Path openAPI, Filter filter, boolea
         String schemaContent = Formatter.format(schemaSyntaxTree).toString();
         if (filter.getTags().size() > 0) {
             // Remove unused records and enums when generating the client by the tags given.
-            schemaContent = modifySchemaContent(schemaSyntaxTree, mainContent, schemaContent, null);
+            schemaContent = GeneratorUtils.modifySchemaContent(schemaSyntaxTree, mainContent, schemaContent, null);
```

Comment:
Shall we consider using a specific name for the util (i.e. `removeUnusedEntities`) instead of using a generic name?

Reply:
+1